makefile.shade

```diff
@@ -16,9 +16,16 @@ k-standard-goals
 #benchmark-tests .compile
   @{
     var sql = string.IsNullOrEmpty(E("BENCHMARK_SQL")) ? string.Empty : "-q " + E("BENCHMARK_SQL");
-    var scenarios = new string[] { "plaintext", "json", "mvcplaintext", "mvcjson" };
+    var scenarios = new string[]
+    {
+      "-n plaintext",
+      "-n plaintext --connectionFilter Benchmarks.PassthroughConnectionFilter",
```

Comment:
camelCase doesn't seem like the right convention - should this be something like `--connection-filter`?

The other thought I had is that it might be better to call it "connection filter type" so it's explicit that's what it does. I could go either way